ci-operator/step-registry/ipi/conf/azurestack/ipi-conf-azurestack-commands.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+REGION="ppe3"
+echo "Azure region: ${REGION}"
+
+workers=3
+if [[ "${SIZE_VARIANT}" == "compact" ]]; then
+  workers=0
+fi
+master_type=null
+if [[ "${SIZE_VARIANT}" == "xlarge" ]]; then
+  master_type=Standard_D32_v3
+elif [[ "${SIZE_VARIANT}" == "large" ]]; then
+  master_type=Standard_D16_v3
+elif [[ "${SIZE_VARIANT}" == "compact" ]]; then
+  master_type=Standard_D8_v3
+fi
+echo $master_type
+
+ENDPOINT="https://management.ppe3.stackpoc.com"
+echo "ASH ARM Endpoint: ${ENDPOINT}"
+
+azurestackjson=$(cat <<EOF
+{
+  "name": "AzureStackCloud",
+  "resourceManagerEndpoint": "https://management.ppe3.stackpoc.com",
+  "activeDirectoryEndpoint": "https://login.microsoftonline.com/",
+  "galleryEndpoint": "https://providers.ppe3.local:30016/",
+  "storageEndpointSuffix": "https://providers.ppe3.local:30016/",
+  "serviceManagementEndpoint": "https://management.stackpoc.com/81c9b804-ec9e-4b5a-8845-1d197268b1f5",
+  "graphEndpoint":                "https://graph.windows.net/",
+  "resourceIdentifiers": {
+    "graph": "https://graph.windows.net/"
+  }
+}
+EOF
+)
+
+echo $azurestackjson >> "${SHARED_DIR}/azurestackcloud.json"
+
+export AZURE_ENVIRONMENT_FILEPATH=${SHARED_DIR}/azurestackcloud.json
+export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=quay.io/padillon/ash-release-mirror:latest
```

Comment:
official release images work for ash now, so this should be removed so that we are testing against the release image created by the pr we are testing against